RichardSkillsLibrary.py

```diff
@@ -74,4 +74,4 @@ def under_bar_and_dance():
         Crabot.move_forward(390)
 
     def speed_and_power(distance):
-        Crabot.gyro_drive(2000, 0, distance)
\ No newline at end of file
+        Crabot.gyro_drive(200000000000000000, 0, distance)
```

Comment:
ummmm

Reply:
I don't know if that is a good idea but it's just in Richard's skill library so ok.